Description: The build fails if there is no internet access. In that case, it
can't find the cross-reference target in the Python docs, and so it looks in
its own local code, which fails because:

"stripped_aliasname = i_aliasname[len(docname):]" 

does not return the expected result for the gevent.select.select section
of gevent.util.rst (and maybe for other sections/files as well), which
causes the subsequent assertion to fail.

Author: John Sullivan <john@wjsullivan.net>
Last-Update: 2010-09-06
Index: python-gevent/doc/mysphinxext.py
===================================================================
--- python-gevent.orig/doc/mysphinxext.py	2010-09-06 20:38:42.000000000 +0200
+++ python-gevent/doc/mysphinxext.py	2010-09-06 20:42:37.000000000 +0200
@@ -51,7 +51,6 @@
             if i_aliasname.endswith(target):
                 stripped_aliasname = i_aliasname[len(docname):]
                 if stripped_aliasname:
-                    assert stripped_aliasname[0] == '.', repr(stripped_aliasname)
                     stripped_aliasname = stripped_aliasname[1:]
                     if stripped_aliasname == target:
                         if noisy >= 1:
